tests/test-basic-user-only.sh

```diff
@@ -21,5 +21,7 @@ set -euo pipefail
 
 . $(dirname $0)/libtest.sh
 
+skip_without_user_xattrs
```

Comment:
The problem is actually because both `setup_exampleos_com` in `libtest.sh` and `basic-test.sh` end up creating `bare-user` repos. The initial part of this one does succeed.